core/src/main/scala/chisel3/aop/Aspect.scala

```diff
@@ -12,6 +12,7 @@ import firrtl.AnnotationSeq
   * @tparam T Type of top-level module
   */
 abstract class Aspect[T <: RawModule] extends Annotation with Unserializable with NoTargetAnnotation {
+  var annotations: AnnotationSeq = Seq()
```

Comment:
Do you think we should make this private, and provide a different accessor?

Reply:
Yes